src/EPROMStore.cpp

```diff
@@ -412,7 +413,7 @@ float EEPROMStore::getRAStepsPerDegree()
 void EEPROMStore::storeRAStepsPerDegree(float raStepsPerDegree)
 {
     // Store steps as 100x steps/deg at 256 MS.
-    const float factor = SteppingStorageNormalized / RA_TRACKING_MICROSTEPPING;
+    const float factor = SteppingStorageNormalized / RA_SLEW_MICROSTEPPING;
```

Comment:
Should add a note about erasing the EEPROM to the changelog (because the storage format is being changed)

Reply:
Will do